### ISSUE TITLE
Add deprecation warnings to deprecated Commerce methods

### ISSUE THREAD

**Issue by thomas-alrek** [apility/netflex-sdk#3]
**Note: This repository's issues are reserved for feature requests and bug reports.**

* **I'm submitting a ...**
  - [ ] bug report
  - [x] feature request
  - [ ] support request => Please do not submit support request here

* **What is the current behavior?**
Deprecated Commerce methods is not marked as deprecated.

* **What is the expected behavior?**
All deprecated methods should be marked as deprecated through Docblocks.

```php
/**
 * @deprecated 1.0.1
 */
public function get_order ($id) {
  /* ... */
}
```

* **What is the motivation / use case for changing the behavior?**
We should not encourage deprecated methods to be used anymore.
And in release 1.1.0 we will remove them from the SDK.

* **Please tell us about your environment:**

  - Package version: 1.0.0
  - PHP version: 7.2

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

https://docs.phpdoc.org/references/phpdoc/tags/deprecated.html

**Comment by thomas-alrek** [comment]
@olavoyen Can you provide a list of methods that should be deprecated?

**Comment by thomas-alrek** [comment]
This will be fixed in milestone v2.0
